source/_meta/_00-head.twig

```diff
@@ -5,7 +5,7 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width" />
     
-    <link rel="stylesheet" href="../../css/style.css?{{ cacheBuster }}" media="all" />
+    <link rel="stylesheet" href="../../css/all.css?{{ cacheBuster }}" media="all" />
```

Comment:
@davidcmoulton Unrelated, but this file contains paths to the fonts (haven't been updated).

Reply:
Good catch, thanks.

Reply:
@thewilkybarkid I've got a fix for the font paths, and also this css file name update in https://github.com/libero/patterns-core/pull/23: a minor PR that just fixes up a few things.